scanner/image/db/scripts/docker-entrypoint.sh

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/env bash
```

Comment:
This looks like it's more based on the entrypoint that was used for scanner instead of the entrypoint for central-db. Is there a reason why this is preferred?

Reply:
This is more like a copy/paste of https://github.com/docker-library/postgres/blob/master/13/bullseye/docker-entrypoint.sh with no difference other than a comment or two.

The only actual difference I see between scanner-db and central-db is that central-db has this:
```
shutdown() {
  pg_ctl -D /var/lib/postgresql/data/pgdata stop -m fast
}

trap shutdown SIGINT SIGTERM

...

	"$@" &
	child=$!
	echo "Waiting for child process $child to exit"
	wait "$child"
```

I can add these things to scanner-db

Reply:
https://github.com/stackrox/stackrox/pull/7269